src/library/scala/Predef.scala

```diff
@@ -58,7 +58,7 @@ import scala.io.StdIn
  *  condition fails, then the caller of the function is to blame rather than a
  *  logical error having been made within `addNaturals` itself. `ensures` is a
  *  form of `assert` that declares the guarantee the function is providing with
- *  regards to it's return value.
+ *  regards to its return value.
```

Comment:
Ouch, I just had to read this one again because it isn't merged yet. Now I have to go use eye drops.